torch/distributed/c10d_logger.py

```diff
@@ -40,30 +41,37 @@ def _get_logging_handler(destination: str = "default") -> Tuple[logging.Handler,
 _c10d_logger = _get_or_create_logger()
 
 
+def _get_msg_dict(func_name, *args, **kwargs) -> Dict[str, Any]:
+    if dist.is_initialized():
+        msg_dict = {
+            "func_name": f"{func_name}",
+            "args": f"{args}, {kwargs}",
+            "pg_name": f"{dist._get_process_group_name(kwargs.get('pg'))}",  # type: ignore[arg-type]
```

Comment:
what will the pg_name look like?

Reply:
pg_name is derived from here: https://fburl.com/7rqfahug